eZ/Publish/Core/FieldType/RichText/Type.php

```diff
@@ -50,25 +50,33 @@ class Type extends FieldType
      */
     protected $internalLinkValidator;
 
+    /**
+     * @var null|\eZ\Publish\Core\FieldType\RichText\CustomTagsValidator
+     */
+    private $customTagsValidator;
+
     /**
      * @param \eZ\Publish\Core\FieldType\RichText\Validator $internalFormatValidator
      * @param \eZ\Publish\Core\FieldType\RichText\ConverterDispatcher $inputConverterDispatcher
      * @param null|\eZ\Publish\Core\FieldType\RichText\Normalizer $inputNormalizer
      * @param null|\eZ\Publish\Core\FieldType\RichText\ValidatorDispatcher $inputValidatorDispatcher
      * @param null|\eZ\Publish\Core\FieldType\RichText\InternalLinkValidator $internalLinkValidator
+     * @param null|\eZ\Publish\Core\FieldType\RichText\CustomTagsValidator|null $customTagsValidator
```

Comment:
Nitpick: `null` is annotated twice

Reply:
I blame PhpStorm ;) 
Amended in 24c37db as well